pkg/kp/kv.go

```diff
@@ -171,3 +222,16 @@ func (c consulStore) Ping() error {
 	}
 	return nil
 }
+
+func addTimeStamp(value WatchResult) (time.Time, WatchResult) {
+	currentTime := time.Now()
+	if value.Time == "" {
+		value.Time = currentTime.String()
+	}
+
+	return currentTime, value
+}
+
+func healthPath(service, node string) string {
+	return fmt.Sprintf("%s%s/%s", service, "health", node)
```

Comment:
hmm ok. I was asking because I was wondering if we would set an ACL on the entire tree (like we have `intent/` and `reality/` and it seems harder to do unless they're all organized under `/health/<service>/<node>` or something, but if y'all are ok with it then can ship